### PR TITLE
chore: multiple style fixes

### PR DESCRIPTION
**Pull request by AlfreMK**
### 🛠 Changes being made

This pr aims to fix some issues:
- (Search page) isDeleted toggle switch was not reactive to changes if a chip related to it in the active filters section was removed.
- (All pages) Admin panel button wasn't being shown after an admin logged in (before you have to reload the window or visit another page to make it appear)
- (Search page) It wasn't displaying correctly the select item text "number of reviews". It was shown "number of re..." instead. Captura de pantalla 2024-07-07 a la(s) 17 06 45
- (Book detail) Other improvements in texts of some components.

Also added a tooltip to the sort order button
Captura de pantalla 2024-07-07 a la(s) 17 13 57

### 📋 Requirements

None

### 🏎 Quality check

- [x] Are your changes following SOLID principles?

- [x] Did you make sure that there aren't any erroneous console logs, debuggers or leftover code in your changes?

- [x] Walk away, take a break, re-read what you filled out above does it make sense if you were coming in cold? What extra context could you provide?

### CODE REVIEW

File: src/pages/search/filters/SearchFiltersSortBy.tsx [hunk 1]
```diff
@@ -33,16 +32,18 @@ export default function SearchFiltersSortBy() {
...
+          className={isDesc ? '' : 'rotate-180'}
+          onClick={() => setIsDesc(!isDesc)}
+          isIconOnly
+        >
+          <ArrowDownIcon />
+        </Button>
+      </Tooltip>
       <Select
-        className="w-[150px]"
+        className="w-[180px]"
```
Comment (mpbascunan) [inline src/pages/search/filters/SearchFiltersSortBy.tsx:46]: [SUG] why can't this be w-fit or something more related to the content?

**Review by mpbascunan — APPROVED** [review]
Looks great! Thanks for these improvements, I just left a question. Also, I think you could add these changes as tech debt cards in Trello (and obviously move them to the Done column once this PR is merged) 👀

File: src/pages/search/filters/SearchFiltersSortBy.tsx (hunk 1 above)
Comment (AlfreMK) [reply src/pages/search/filters/SearchFiltersSortBy.tsx:46]: I tried that at first but after some attempts getting this:
Captura de pantalla 2024-07-07 a la(s) 17 47 30
I just gave up XD. Maybe there is some tricks with css (or changing parent component styles) that will achieve solving it in a responsive way but I don't it is worth the time

**Comment by AlfreMK** [comment]
> Looks great! Thanks for these improvements, I just left a question. Also, I think you could add these changes as tech debt cards in Trello (and obviously move them to the Done column once this PR is merged) 👀

Good idea!, I will add them as tech debt

File: src/pages/search/filters/SearchFiltersSortBy.tsx (hunk 1 above)
Comment (mpbascunan) [reply src/pages/search/filters/SearchFiltersSortBy.tsx:46]: jaja I see, I thought it was easier, but I agree with you

**Pull request merged**